controllers/users-controller.js

```diff
@@ -73,7 +73,7 @@ exports.registerUser = (appReq, appRes) => {
       });
 
       appRes.status(200).send({ auth: true, token });
-    },
```

Comment:
These are correct per eslint rules - personally I agree with I think that that rule is odd but it doesn't break anything so we should follow it for consistent style across developers. I haven't had a chance to dig through eslint rules further to see what is driving that decision maybe is part of the AirBnB style which we're following. If you like feel free to review the lint rules and start a conversation on things you think we should change; keep in mind, we want to use the same code style for both front and back end. Thanks

Reply:
yep ditto on what @jkwening said. I think it is just an easiest means of consistency for airbnb, operating with such large teams it is a lot easier to set a rule that says always do xyz than to try and specify in theses cases do xyz. 
With that aside everything else about the your set up and the PR looks good. I will close this pr for now unless we end up changing lint settings.
I think you should be good to set up the docs directory and start on a strapdown index file.
Thanks!